theta: [1.570796327, 3.141592654, 0]
alpha: [0, 0, 0]
d: [0, 0, 0]
a: [0.094, 0.06107, 0.04903524]
origin:
  left_arm: [0, -0.1583424671, -0.07349, 0, 0, 0]
  right_arm: [0, 0.1583424671, -0.07349, 0, 0, 3.14]